pennylane_lightning/lightning_qubit/__init__.py

```diff
@@ -1,4 +1,4 @@
-# Copyright 2018-2023 Xanadu Quantum Technologies Inc.
+# Copyright 2018-2024 Xanadu Quantum Technologies Inc.
```

Comment:
For ref, we don't need to update these on PR changes, as the original date carries forward :)

Reply:
In this case, I propose we move to delete the second year gradually. 
I think this would make things clear for everyone.